### PR TITLE
Added profile screen design

### PR DESCRIPTION
**Pull request by nikunjdhamelia**
- Added profile screen design.
- Added Statusbar

### CODE REVIEW

File: app/components/ProfileMenuList/index.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { View, FlatList, TouchableOpacity, Text } from "react-native";
+import {normalize} from '../../helper/responsiveScreen';
```
Comment (Spielberg) [inline app/components/ProfileMenuList/index.tsx:3]: @nikunjdhamelia, please change those imports; we always use absolute paths

File: app/components/ProfileMenuList/index.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { View, FlatList, TouchableOpacity, Text } from "react-native";
```
Comment (Spielberg) [inline app/components/ProfileMenuList/index.tsx:2]: @nikunjdhamelia don't mix between " and ' use always the single one

File: app/components/ProfileMenuList/index.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+import {normalize} from '../../helper/responsiveScreen';
+import colors from '../../assets/colors';
+import {FontText} from '..';
+import SvgIcons from '../../assets/svgIcons';
+import styles from './style';
+
+const ProfileMenuList = (props: any) => {
+  const renderItem = ({ item, index }: any) => (
```
Comment (Spielberg) [inline app/components/ProfileMenuList/index.tsx:10]: @nikunjdhamelia move this to a new component `Item`, in the same file with a parameter `isBorderDisplayed` please.

File: app/containers/Profile/index.tsx [hunk 4]
```diff
@@ -1,29 +1,73 @@
...
+import { FontText } from "@/app/components";
+import colors from "@/app/assets/colors";
+import { hp, normalize } from "@/app/helper/responsiveScreen";
+import ProfileMenuList from "@/app/components/ProfileMenuList";
+import { PROFILE_MENU_DATA } from "@/app/helper/constantData";
 
-const Profile = ({navigation}: any) => {
+const Profile = ({ navigation }: any) => {
   const { t } = useTranslation();
+  const [menu, setMenu] = React.useState<any>([]);
```
Comment (Spielberg) [inline app/containers/Profile/index.tsx:14]: @nikunjdhamelia why we use any?

File: app/helper/constantData.ts [hunk 5]
```diff
@@ -61,3 +61,46 @@ export const TREADING_COURSES_DATA = () => {
     },
   ];
 };
+
+
+export const PROFILE_MENU_DATA = () => {
```
Comment (Spielberg) [inline app/helper/constantData.ts:66]: @nikunjdhamelia love this, good job

File: app/helper/i18n/translation/es.json [hunk 6]
```diff
@@ -40,5 +40,17 @@
   "price": "Price",
 
   "TODO: remove all the above translations and add your own translations": "",
-  "app.register.title": "Register"
+  "app.register.title": "Register",
+
```
Comment (Spielberg) [inline app/helper/i18n/translation/es.json:44]: @nikunjdhamelia remove empty lines

File: app/components/ProfileMenuList/index.tsx (hunk 1 above)
Comment (nikunjdhamelia) [reply app/components/ProfileMenuList/index.tsx:3]: Okay, it's done.

File: app/components/ProfileMenuList/index.tsx (hunk 2 above)
Comment (nikunjdhamelia) [reply app/components/ProfileMenuList/index.tsx:2]: It's done

File: app/components/ProfileMenuList/index.tsx (hunk 3 above)
Comment (nikunjdhamelia) [reply app/components/ProfileMenuList/index.tsx:10]: It's done.
I have created a separate component.

File: app/containers/Profile/index.tsx (hunk 4 above)
Comment (nikunjdhamelia) [reply app/containers/Profile/index.tsx:14]: Sorry for that. I have added this type of array.

File: app/helper/constantData.ts (hunk 5 above)
Comment (nikunjdhamelia) [reply app/helper/constantData.ts:66]: Thanks.

**Pull request merged**